src/monitor/health_check_worker.c

```diff
@@ -45,16 +45,9 @@
 #include "utils/memutils.h"
 #include "tcop/utility.h"
 
-/*
- * The healthcheck only checks if it gets a response back from postgres. So
- * both user and password are actually useless, because we do not need to
- * authenticate. They are provided though to override any settings set through
- * PGPASSWORD environment variable or .pgpass file. This way it does not matter
- * that TLS is not necessarily used, because no secret information is sent.
```

Comment:
Let's keep part of this comment

Reply:
Oops, good point. I guess the git revert killed that comment. Re-installed now!